### ISSUE TITLE
The Total pricing field is missing from the Edit Entry layout

### ISSUE THREAD

**Issue by rafaehlers** [GravityKit/GravityView#2230]
Here's my form:

![image](https://github.com/user-attachments/assets/e8f2c9f5-6a3b-4223-b2e8-1602fda23337)

Here's my Edit Entry layout. Notice how the Total field is missing:

![image](https://github.com/user-attachments/assets/a23538e4-7fb5-4934-9d47-6b66abf724a3)

The field does show up if no fields were added to the Edit Entry layout:

![image](https://github.com/user-attachments/assets/82cb1be6-f253-4012-97d8-cb9b617b8dc2)

The field also shows if adding its Merge Tag inside a Custom Content field.

Video: https://www.loom.com/share/1c5dd06da43e4bcba21a7eaa77d231bd

Tested on GV 2.32 and GF 2.9

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/2781156800/59697

**Comment by omarkasem** [comment]
@rafaehlers It's done on purpose and here's the explanation, Should we change that and maybe add a notice to the user that he has to include all fields or keep it like that?
![image](https://github.com/user-attachments/assets/d5ac638e-9a9f-4070-86c2-7eaf9fe7caf7)

**Comment by rafaehlers** [comment]
@omarkasem Thanks for clarifying it!

@zackkatz any tips on how to proceed here? 🤔 This one is tricky!

**Comment by zackkatz** [comment]
@omarkasem Right. Now that I've had (looks at calendar) 10 years to think about this: could we use AJAX to update it, sending the field data for fields that are shown and returning new value?

**Comment by omarkasem** [comment]
@zackkatz I tried that but what if there are more than 1 product field, which one we will choose to get final values from? Or we get from all products fields?
I don't think it makes sense to do it that way as why would the client only have a total field on it's own in the edit page? He can't edit it.
I think the best thing to do is to let him know that he has to choose at least 1 product field so the total can work, Add a tooltip warning in the dashboard when he adds a total field to the edit layout and also add one in frontend on top of total field
